### PR TITLE
fix playbook permissions

### PR DESCRIPTION
**Pull request by bloodearnest**

### CODE REVIEW

File: scripts/jobrunner.sh [hunk 1]
```diff
@@ -85,8 +85,11 @@ test -f ~jobrunner/.bashrc || touch ~jobrunner/.bashrc
 grep -q "jobrunner/bashrc" ~jobrunner/.bashrc || echo 'test -f /srv/jobrunner/bashrc && . /srv/jobrunner/bashrc' >> ~jobrunner/.bashrc
 
 # update playbook
-cp jobrunner/playbook.md /srv/playbook.md
-ln -sf "/srv/playbook.md" ~jobrunner/playbook.md
+cp jobrunner/playbook.md /srv/jobrunner/playbook.md
+ln -sf "/srv/jobrunner/playbook.md" ~jobrunner/playbook.md
```
Comment (ghickman) [inline scripts/jobrunner.sh:89]: what's the `~jobrunner` syntax here?

**Review by ghickman — APPROVED** [review]

File: scripts/jobrunner.sh (hunk 1 above)
Comment (bloodearnest) [reply scripts/jobrunner.sh:89]: home directory of user `jobrunner`

Comment (ghickman) [reply scripts/jobrunner.sh:89]: TIL, thanks 👍

File: scripts/jobrunner.sh [hunk 2]
```diff
@@ -85,8 +85,11 @@ test -f ~jobrunner/.bashrc || touch ~jobrunner/.bashrc
...
+
+# clean up old playbook if present
+rm -rf /srv/playbook.md
```
Comment (madwort) [inline scripts/jobrunner.sh:92]: not sure you need `-r` here!

Comment (bloodearnest) [reply scripts/jobrunner.sh:92]: fixed

**Comment by bloodearnest** [comment]
wat. I didn't close it

**Pull request closed without merging**